client/src/reducers/controls.ts

```diff
@@ -37,13 +39,31 @@ const minimizeMaximizePopUps = (
   };
 };
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
+interface ControlsState {
+  loading: boolean;
+  error: Error | string | null;
+  resettingInterface: boolean;
+  graphInteractionMode: string;
+  opacityForDeselectedCells: number;
+  scatterplotXXaccessor: string | boolean;
+  scatterplotYYaccessor: string | boolean;
+  geneIsOpen: boolean;
+  scatterplotIsMinimized: boolean;
+  geneIsMinimized: boolean;
+  scatterplotLevel: string;
```

Comment:
this one possibly just the following too:

```suggestion
  scatterplotLevel: "top" | "bottom";
```